airflow/example_dags/tutorial_xcom_pandas.py

```diff
@@ -0,0 +1,149 @@
+#
```

Comment:
Any suggestion on where this file should live @turbaszek?

It can stay here too -- If we decide to keep it here, we need to apply the following diff as one of the tests is failing:

```diff
diff --git a/tests/jobs/test_scheduler_job.py b/tests/jobs/test_scheduler_job.py
index da27016a0..f8b6f3811 100644
--- a/tests/jobs/test_scheduler_job.py
+++ b/tests/jobs/test_scheduler_job.py
@@ -3235,6 +3235,7 @@ class TestSchedulerJob(unittest.TestCase):

         ignored_files = {
             'helper.py',
+            'tutorial_xcom_pandas.py',
         }
         example_dag_folder = airflow.example_dags.__path__[0]
         for root, _, files in os.walk(example_dag_folder):  # pylint: disable=too-many-nested-blocks
```

Reply:
@kaxil there was discussion on devlist and I think we've never reached consensus. But in general we were against having custom XCom backend in code, but we may revisit it.

Reply:
@turbaszek It is possible that I misunderstood the email thread on the devlist, but I thought the conclusion from that thread was that including examples of custom XCom backend is good. And that is exactly what this attempts to be - an example.

Reply:
@vikramkoka I have no strong opinion. I would only consider adding `.airflowignore` to avoid parsing non-dag files in example directories. Also if the example is there in code, we should make sure that it can be used by anyone.